students/olszewski_bartosz/lesson_01_python_basics/digital_clock.py

```diff
@@ -0,0 +1,6 @@
+print ('Podaj liczbę minut')
+a = int(input())
+a = a%1440
+godziny = a//60
+minuty = a - godziny*60
+print('Godzina wskazana na zegarze to',godziny,':',minuty)
```

Comment:
Please ensure that you can pass all Snakify tests. After you are logged in, there is available button 'Save and check the solution on all tests'
BTW, great that even you haven't check the Snakify tests the logic is good.

Try to use format in print statements instead of comma
https://pyformat.info/
print('Godzina wskazana na zegarze to {}:{}'.format(godziny, minuty